### DISCUSSION TITLE
Can someone copy your docusaurus source md files ?

### DISCUSSION THREAD

**Discussion by mem212** [facebook/docusaurus#5744] [Q&A] [answered]
I understand docusaurus main objective is to provide content to public or to a restircted community.

I want to publish some content on the Internet ( public), however I don't want people to just copy my source files ( .md files) and publish the same content using another docusaurus website.

Is it possible with docusaurus to protect my source files ? 

Thanks

**Comment by slorber** [comment, marked as answer]
You can use a private Git repository.

Your source md files are not exposed by Docusaurus, as they are not part of the `website/build` folder

**Reply by mem212** [reply]
That was quick thank you
